packages/fast-usdc/test/fast-usdc.contract.test.ts

```diff
@@ -410,8 +423,8 @@ const makeCustomer = (
       // "cctp" here has some noble stuff mixed in.
       const tx = cctp.makeTx(amount, recipientAddress);
       t.log(who, 'signs CCTP for', amount, 'uusdc w/EUD:', EUD);
-      txPublisher.publish(tx);
-      sent.push(tx);
+      txPublisher.publish({ evidence: tx, isRisk });
```

Comment:
this makes it look like the customer decides when things are risky. I would expect that to be in `makeOracleOperator`.